lib/cdo/shared_constants/pd/teacher1819_application_constants.rb

```diff
@@ -1,5 +1,7 @@
 module Teacher1819ApplicationConstants
   # Remove newlines and leading whitespace from multiline strings
```

Comment:
This comment should be below your constant because it applies to `self.clean_multiline`